oxidation/libparsec/crates/types/src/addr.rs

```diff
@@ -809,11 +808,12 @@ impl BackendInvitationAddr {
         let invitation_type = match extract_action(&pairs)? {
             x if x == "claim_user" => InvitationType::User,
             x if x == "claim_device" => InvitationType::Device,
+            x if x == "claim_shamir_recovery" => InvitationType::ShamirRecovery,
```

Comment:
```suggestion
            "claim_user" => InvitationType::User,
            "claim_device" => InvitationType::Device,
            "claim_shamir_recovery" => InvitationType::ShamirRecovery,
```

Directly using literal is better

Reply:
a865d44fa